README.md

```diff
@@ -34,3 +34,70 @@ env FLASK_APP=python_receive_a_message_tutorial.py flask run
 ## Getting Help
 
 If you are experiencing difficulties, [contact support](https://freeclimb.com/support).
+
+# Python SMS Quickstart
+
+This quickstart serves as a guide to get your first SMS application up and running with [FreeClimb](https://docs.freeclimb.com/docs/how-freeclimb-works).
+
+Specifically, the project will:
+
+- Receive an incoming message via a FreeClimb application
+- Respond "Hello World!" to the incoming message
+
+## Tutorial
+
+We offer a [Python SMS Quickstart Tutorial](https://docs.freeclimb.com/docs/python-messaging-quickstart) for more detailed set-up instructions and explanation of how FreeClimb works.
+
+## Requirements
+A [FreeClimb account](https://www.freeclimb.com/dashboard/signup/)
+
+A [registered application](https://docs.freeclimb.com/docs/registering-and-configuring-an-application#register-an-app) with a named alias
+
+A [configured FreeClimb number](https://docs.freeclimb.com/docs/getting-and-configuring-a-freeclimb-number) assigned to your application
+
+Trial accounts: a [verified number](https://docs.freeclimb.com/docs/using-your-trial-account#verifying-outbound-numbers)
+
+Tools:
+- [Python](https://www.python.org/downloads/) 3.5.0 or higher
+- [Yarn](https://yarnpkg.com/en/)
```

Comment:
pip

Reply:
I think pip comes packaged with python, should I still include it?

Reply:
ya i think python has some alternative package managers so it just makes it clear we're using pip for this project

Reply:
remove yarn